apps/formula1/formula_1.star

```diff
@@ -0,0 +1,280 @@
+
+    elif config.get("local_timezone") == "MST":
+        if MST <= 0:
+            TZ = str(MST + 24) + " MST"
+            DAY = str(RACE_DAY_1)[8:10]
+            ADJ_Month = str(RACE_DAY_1)[5:7]
+        else:
+            TZ = str(MST) + " MST"
+            DAY = F1_DATE[8:10]
+            ADJ_Month = F1_DATE[5:7]
+
+    elif config.get("local_timezone") == "PST":
+        if PST <= 0:
+            TZ = str(PST + 24) + " PST"
+            DAY = str(RACE_DAY_1)[8:10]
+            ADJ_Month = str(RACE_DAY_1)[5:7]
+        else:
+            TZ = str(PST) + " PST"
+            DAY = F1_DATE[8:10]
+            ADJ_Month = F1_DATE[5:7]
+
+    elif EST <= 0:
+        TZ = str(EST + 24) + " EST"
+        DAY = str(RACE_DAY_1)[8:10]
+        ADJ_Month = str(RACE_DAY_1)[5:7]
+    else:
+        TZ = str(EST) + " EST"
+        DAY = F1_DATE[8:10]
+        ADJ_Month = F1_DATE[5:7]
+
+    #find the month and display as text
+    if ADJ_Month == "01":
+        Month = "JAN"
+
+    elif ADJ_Month == "02":
+        Month = "FEB"
+
+    elif ADJ_Month == "03":
+        Month = "MAR"
+
+    elif ADJ_Month == "04":
+        Month = "APR"
+
+    elif ADJ_Month == "05":
+        Month = "MAY"
+
+    elif ADJ_Month == "06":
+        Month = "JUN"
+
+    elif ADJ_Month == "07":
+        Month = "JUL"
+
+    elif ADJ_Month == "08":
+        Month = "AUG"
+
+    elif ADJ_Month == "09":
+        Month = "SEP"
+
+    elif ADJ_Month == "10":
+        Month = "OCT"
+
+    elif ADJ_Month == "11":
+        Month = "NOV"
+
+    else:
+        Month = "DEC"
+
+    #30x24 track maps
+    Abu_Dhabi = base64.decode("iVBORw0KGgoAAAANSUhEUgAAAB4AAAAYCAYAAADtaU2/AAAAAXNSR0IArs4c6QAAAO1JREFUSEvNllsOxSAIRGX/i/YGEwyOiKgkt/3qQz0wDFYqpdTyh4s+D651FIaIY76/QhkLVGD4fIPfgleQV7gL5sVRUv3uBb4EW4vqOr/KboI9eRmIStxkPoEtedk8evFVCXhc1O0DOGqkSHA7p3ewJ5fVTqvMorIPYDSMSGfV1JM0Am9grB8CT8HaE1pyHSwJVWeL9zdgrDG2YgOvetLqW8nGkhr3c8vlXV0ER9rBc7WebwUiSkwZZ4K9lkoDR5w8GE2kPplo/ShOdq02tvfSwXaHtYuUB2Wf2mm31WV93x4EskBTxp8/7GVn/gMjGOn79JCBIwAAAABJRU5ErkJggg==")
+    Spielberg = base64.decode("iVBORw0KGgoAAAANSUhEUgAAAB4AAAAYCAYAAADtaU2/AAAAAXNSR0IArs4c6QAAAS9JREFUSEullisOwkAQhrcHQHEEDEFxjRo4A3UIMG0QNMG1CSEIDLKcAQyX4AIk4NA4BAqyIUuWZTqvVjVtZ77557HTyBjzMsKrqqqvRZIkQuvP55EU7EMdUQMXgSGl7pkUzgbXpbcObJ9jwZBgrJ4YlCoDCqZUWuehKm4PgGDfeFxuh8/Lae8USDNQl5U/MNYsGLTfa+fTbFVAdYV8/oC1UKqzoYBZijkBYR1MKg7r6DuDjCml1h+7xlx4Eyh5ZGIqoVHyD20qMPYBEqZdWtNwk5BgrE7QWqKUOhsWmAvvdlqjWb7ZcRYGG0zBpTtaBMbgFrycZPH5cT9y/gzEYB9u78t1Opin60OxyOLrjQclx4mKXJpe359KcTivnGZSjROlXPO+sWINtHGNtVBr9wYhpcwBXE9y4wAAAABJRU5ErkJggg==")
+    Melbourne = base64.decode("iVBORw0KGgoAAAANSUhEUgAAAB4AAAAYCAYAAADtaU2/AAAAAXNSR0IArs4c6QAAANVJREFUSEvtlkEOhCAMRetmXMwtORi3nIVuxjBJCWBpf1FjYsYdofSV/7EwEdGXbvgmCRxjrEoJIZxe2g7MUIa147MqqMA9yBXwDLaSW/NeJSqw5WWCWzFoAT8wuptngREJL9kxCk4eIrGW19ljNJl2HnhuWd80vz6ZLeV2g1M2CW71gFapIXAJLyXVVGvPh+t3snzT5ltFXA3kCJhVYlX+4KNqquvLAwbfTqMV9R4V4n3cg6BNhtdrzUZ8+khg9AZDoCkGBveahmaBppALPOqztO428AbVhp8B0Iv3ZQAAAABJRU5ErkJggg==")
```

Comment:
It would be tidier if we moved all these map images to a constant at the top of file.
Additionally, instead of matching by locality, we could use the `circuitId` provided by the API. It appears more suitable for our use here.

```
MAPS = dict(
  albert_park = base64(....),
  imola = base64(....),
  interlagos = base64(....)
)

render.Image(src = MAPS.get(circuitId, 'americas'))
```

Reply:
Do i need the americas portion in 

render.Image(src = MAPS.get(circuitId, 'americas'))

or is this okay? 

render.Image(src = MAPS.get(circuitId))

Reply:
If the circuitId is not in the MAPS dict it will return none, so unless you are sure the circuitId is going to always be in that dict you should keep the `americas`  line as then it will avoid render.Image erroring if the map can't be found. 

you may want to deal with that though and have it show map not found though instead of giving the "wrong" map though.